tutorials/onchain-governance/fast-track-gov-proposal.md

```diff
@@ -0,0 +1,222 @@
+	// The code will be added here
+
+	process.exit(0)
+}
+
+--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:try-catch-block'
+```
+
+This structure provides the foundation for your script. It imports all the necessary dependencies and sets up a main function that will contain the core logic of your proposal submission process.
+
+### Connect to the Forked Chain
+
+Inside your `main` function, add the code to connect to your local Polkadot fork:
+
+```typescript hl_lines="2-3"
+--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:main'
+```
+
+???+ function "**connectToFork** ()"
+
+	```typescript
+	--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:connectToFork'
+	```
+
+### Create and Submit the Proposal
+
+In this step, you will perform the following actions:
+
+1. Define the call you want to execute and its origin
+2. Create a [preimage](/polkadot-protocol/glossary#preimage){target=\_blank} using the selected call
+3. Submit the proposal. It uses the preimage hash (obtained from the call) as part of the proposal submission process. The proposal is submitted with the selected origin
+4. Place decision deposit. This deposit is required to move the referendum from the preparing phase to the deciding phase
+
+```typescript hl_lines="5-14"
+--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:main'
+```
+
+!!!note
+    The [`setCodeWithoutChecks`](https://paritytech.github.io/polkadot-sdk/master/frame_system/pallet/struct.Pallet.html#method.set_code_without_checks){target=\_blank} extrinsic used in this example is for demonstration purposes only. Replace it with the specific extrinsic that matches your governance proposal's intended functionality. Ensure the call matches the runtime requirements and governance process of your target Polkadot SDK-based network.
+
+???+ function "**generateProposal** (api, call, origin)"
+
+	The `generateProposal` function accomplishes these tasks using a batched transaction, which combines multiple operations into a single transaction:
+
+    1. `preimage.notePreimage` - this submits the preimage of the proposal
+    2. `referenda.submit` - submits the actual proposal to the referenda system
+    3. `referenda.placeDecisionDeposit` - places the required decision deposit for the referendum
+   
+	```typescript
+	--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:generateProposal'
+	```
+
+### Force Proposal Execution
+
+After submitting your proposal, you may want to test its execution without waiting for the standard voting and enactment periods. Chopsticks allows you to force the execution of a proposal by manipulating the chain state and the scheduler.
+
+```typescript hl_lines="16-17"
+--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:main'
+```
+
+???+ function "**forceProposalExecution** (api, call, origin)"
```

Comment:
Is there a separate mkdocs PR that goes with this one? Right now these are all rendering as note admonitions, which I actually didn't really notice until I saw this one which has a note inside of it. So we should be using something different for these

Reply:
There's no PR in mkdocs. What do you suggest we can do to improve this?

Reply:
@eshaben : child admonitions instead?

Reply:
No, not child admonitions cause those will look weird since they take up the width of the text inside them so they have the potential to be all different sizes

Reply:
Would have to come up with something new, but since noticing this, I've also noticed other places were defaulting to the note. So it's fine for now and we can circle back to changing these in a separate PR that fixes all of the instances in one swoop